lib/board.rb

```diff
@@ -0,0 +1,16 @@
+class Board
+  attr_reader :board
+
+  def initialize
+    @board = [1, 2, 3, 4, 5, 6, 7, 8, 9]
+  end
+
+  def [](num)
+    @board[num]
+  end
+
```

Comment:
Just like you have place_mark below, can we have a more descriptive name for this method?  For example, @board.place_mark tells me exactly what it's doing.  board[0] exposes the data structure and in plain english does not tell me what it's doing.

Reply:
Ok, renamed this to find_position